packages/app/cypress/integration/ta/student_question.spec.js

```diff
@@ -25,37 +36,39 @@ describe("TA interacts with student question", () => {
     );
   });
 
-  it("clicks the help button then finish helping", () => {  
+  it("clicks the help button then finish helping", () => {
     // Click on the student's question
-    
+    cy.get("[data-cy='ta-queue-card']").should("be.visible").click();
     // Click help
+    cy.get("[data-cy='help-student']").click();
 
     // Click Finish Helping
-
+    cy.contains("button", "Finish Helping").click();
   });
 
   it("clicks help button then mark question as can't find", () => {
     // Click on the student's question
-
+    cy.get("[data-cy='ta-queue-card']").should("be.visible").click();
     // Click help
+    cy.get("[data-cy='help-student']").click();
 
-    // Click Can't Helping
-
+    // Click Finish Helping
```

Comment:
Did you mean for this to be 
```suggestion
    // Click Can't Find
```